app/scripts/components/common/map/style-generators/raster-paint-layer.tsx

```diff
@@ -73,14 +77,22 @@ export function RasterPaintLayer(props: RasterPaintLayerProps) {
       async function run() {
         // Create a modified version of the parameters
         const tileParamsAsString = formatTitilerParameter(updatedTileParams);
-        const tileUrl = `${tileApiEndpoint}?${tileParamsAsString}`;
+        const tileUrl = Array.isArray(tileApiEndpoint)
```

Comment:
I think you should be able to handle this on parent level, by passing `sourceParamFormatter`. It will be best if raster-paint-layer can stay agnostic to any layer.